.github/workflows/pr.yml

```
@@ -133,7 +133,7 @@ jobs:
 
   publish:
     needs: [checks, unit_tests, pvc_unit_tests, ishmem_unit_tests]
-    runs-on: cpu
+    runs-on: pvc
```

```suggestion
    runs-on: intel-ubuntu-latest
```

Just run the unit tests on the cluster. These other jobs are ok for the regular public resources